Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.7-slim-buster
+LABEL maintainer="Aspergillus Developers <aspergillus@mail.computational-biology.org>"
+
+WORKDIR /nlisim
+COPY . /nlisim
+
+RUN apt update \
+	&& apt install -y git libgl1-mesa-glx \
+	&& pip3 install --upgrade pip \
+	&& pip install --editable .
```

Comment:
Is it expected that users re-mount this code while running the container? If not, I wouldn't install as editable.

Reply:
Removed